python/qpid_dispatch/management/error.py

```diff
@@ -114,27 +114,27 @@ def __init__(self, description): ManagementError.__init__(self, status, descript
     return Error
 
 
-class BadRequestStatus(_error_class(BAD_REQUEST)):
+class BadRequestStatus(_error_class(BAD_REQUEST)):  # type: ignore[misc]  #  Unsupported dynamic base class "_error_class"
```

Comment:
This is how you suppress mypy line per line. Here, the _static_ type checker protests against us inheriting from a _dynamically_ determined class. Makes sense and obviously needs a suppression.